en/orm/behaviors/counter-cache.rst

```diff
@@ -105,7 +105,10 @@ Your function can return ``false`` to skip updating the counter column, or
 a ``Query`` object that produced the count value. If you return a ``Query``
 object, your query will be used as a subquery in the update statement.  The
 ``$table`` parameter refers to the table object holding the behavior (not the
-target relation) for convenience.
+target relation) for convenience. The callback is invoked at lease once were
```

Comment:
Should 'lease' be 'least' here, and 'were' be 'where'?

Reply:
fixed